data-service-client/src/main/java/com/amazonaws/blox/dataserviceclient/v1/client/DataServiceLambdaClientConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataserviceclient.v1.client;
+
+import com.amazonaws.blox.dataservicemodel.v1.client.DataService;
+import com.amazonaws.blox.dataservicemodel.v1.serialization.DataServiceMapperFactory;
+import com.amazonaws.blox.jsonrpc.JsonRpcLambdaClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import software.amazon.awssdk.services.lambda.LambdaAsyncClient;
+
+/** AWS Lambda client for the DataService using JSONRPC for routing. */
+@Configuration
+public class DataServiceLambdaClientConfiguration {
```

Comment:
Is this intended only for integration tests? If we model it as a Spring configuration class, I don't think the `@Import` for this will work correctly in e.g. the scheduling manager, since it also defines a lambda client bean, and it gets its function name from an environment variable rather than from CloudFormation. Perhaps this can be only a factory method instead?